sklearn/kernel_ridge.py

```diff
@@ -192,3 +182,234 @@ def predict(self, X):
+
+    coef0 : float, default=1
+        Zero coefficient for polynomial and sigmoid kernels.
+        Ignored by other kernels.
+
+    kernel_params : mapping of string to any, optional
+        Additional parameters (keyword arguments) for kernel function passed
+        as callable object.
+
+    Attributes
+    ----------
+    dual_coef_ : array, shape = [n_samples] or [n_samples, n_targets]
+        Representation of weight vector(s) in kernel space
+
+    X_fit_ : {array-like, sparse matrix}, shape = [n_samples, n_features]
+        Training data, which is also required for prediction
+
+    References
+    ----------
+    * Kevin P. Murphy
+      "Machine Learning: A Probabilistic Perspective", The MIT Press
+      chapter 14.4.3, pp. 492-493
+
+    See also
+    --------
+    sklearn.linear_model.Ridge:
+        Linear ridge regression.
+    sklearn.svm.SVR:
+        Support Vector Regression implemented using libsvm.
+    sklearn.kernel_ridge.KernelRidgeClassifier:
+        Kernel Ridge implemented for multiclass clasifications.
+
+    Examples
+    --------
+    >>> from sklearn.kernel_ridge import KernelRidge
+    >>> import numpy as np
+    >>> n_samples, n_features = 10, 5
+    >>> rng = np.random.RandomState(0)
+    >>> y = rng.randn(n_samples)
+    >>> X = rng.randn(n_samples, n_features)
+    >>> clf = KernelRidge(alpha=1.0)
+    >>> clf.fit(X, y) # doctest: +NORMALIZE_WHITESPACE
+    KernelRidge(alpha=1.0, coef0=1, degree=3,
+                gamma=None, kernel='linear', kernel_params=None)
+    """
+    pass
+
+
+class KernelRidgeClassifier(ClassifierMixin, _BaseKernelRidge):
+    """Kernel Ridge Classifier.
+
+    Kernel Ridge Classifier (KRC) combines ridge
+    regression (linear least squares with l2-norm regularization), special
+    encoding for target and the kernel trick. It thus
+    learns a linear function in the space induced by the respective kernel and
+    the data. For non-linear kernels, this corresponds to a non-linear
+    function in the original space.
+
+    The form of the model learned by KRC is identical to support vector
+    classification (SVC). However, different loss functions are used:
+    KRR uses squared error loss while support vector classification uses
+    epsilon-insensitive loss, both combined with l2 regularization.
+    In contrast to SVC, fitting a multiclass KRC model can be done
+    at once, without using one-against-one or one-against-rest techniques.
+
+    Read more in the :ref:`User Guide <kernel_ridge>`.
+
+    Parameters
+    ----------
+    alpha : {float, array-like}, shape = [n_targets]
+        Small positive values of alpha improve the conditioning of the problem
+        and reduce the variance of the estimates.  Alpha corresponds to
+        ``(2*C)^-1`` in other linear models such as LogisticRegression or
+        LinearSVC. If an array is passed, penalties are assumed to be specific
+        to the targets. Hence they must correspond in number.
```

Comment:
Maybe add a generic sentence such as:

High values of alpha will tend to make the learned decision function smoother but potentially cause underfitting while smaller values of alpha will preserve the influence of the individual training datapoints and therefore overfitting although this highly depends on the choice of the kernel parameters.